mcux/CMakeLists.txt

```diff
@@ -61,6 +61,16 @@ add_subdirectory_ifdef(
   middleware/wireless/framework_5.3.3
   )
 
+if (${MCUX_DEVICE} MATCHES "MIMX8MN6")
+	zephyr_library_sources_ifdef(CONFIG_IPM_IMX_MCUX	drivers/imx8/fsl_mu.c)
+	zephyr_library_sources(drivers/imx8/fsl_gpc.c)
+	zephyr_library_sources(devices/${MCUX_DEVICE}/system_MIMX8MN6_cm7.c)
+	add_subdirectory_ifdef(
```

Comment:
remove the rpmsg_lite reference here

Reply:
The drivers handling here needs to move to `mcux/drivers/imx8/CmakeLists.txt`